### PR TITLE
Announcing Scala.js 0.6.33.

### PR DESCRIPTION
**Pull request by sjrd**

### CODE REVIEW

File: _posts/news/2020-05-13-announcing-scalajs-0.6.33.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+category: news
+tags: [releases]
+permalink: /news/2020/05/13/announcing-scalajs-0.6.33/
+---
+
+
+We are pleased to announce the release of Scala.js 0.6.33!
+
+This is mostly a bugfix release, including a fix for a bad interaction with React.js' development mode and the `jsdom` JS env.
+In addition, we have upgrade to the standard libraries of Scala 2.12.11 and 2.13.2.
```
Comment (gzm0) [inline _posts/news/2020-05-13-announcing-scalajs-0.6.33.md:13]: upgrade*d*

File: _posts/news/2020-05-13-announcing-scalajs-0.6.33.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+title: Announcing Scala.js 0.6.33
+category: news
+tags: [releases]
+permalink: /news/2020/05/13/announcing-scalajs-0.6.33/
+---
+
+
+We are pleased to announce the release of Scala.js 0.6.33!
+
+This is mostly a bugfix release, including a fix for a bad interaction with React.js' development mode and the `jsdom` JS env.
```
Comment (gzm0) [inline _posts/news/2020-05-13-announcing-scalajs-0.6.33.md:12]: 0.6.33 will start to warn, if one overrides `equals` or `hashCode` in a `js.Any` (scala-js/scala-js#3969). Should we talk about it here?

File: _posts/news/2020-05-13-announcing-scalajs-0.6.33.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+Please report any issues [on GitHub](https://github.com/scala-js/scala-js/issues).
+
+## Bug fixes
+
+Among others, the following bugs have been fixed in 0.6.33:
+
+* [#4034](https://github.com/scala-js/scala-js/issues/4034) Incorrect result for `-x` when `x` is `+0.0`
+* [#3458](https://github.com/scala-js/scala-js/issues/3458) Bad interactions in `JSDOMNodeJSEnv` error handling with React.js dev mode
+* [#3818](https://github.com/scala-js/scala-js/issues/3818) Linking error for `Future.never` from Scala 2.13
+* [#3939](https://github.com/scala-js/scala-js/issues/3939) Compile error on a method with `@JSName("finalize")` in a non-native JS class
```
Comment (gzm0) [inline _posts/news/2020-05-13-announcing-scalajs-0.6.33.md:43]: https://github.com/scala-js/scala-js/issues/3998 deliberately not mentioned?

**Comment by sjrd** [comment]
Thanks for the review. Those are all good points. Updated.

File: _posts/news/2020-05-13-announcing-scalajs-0.6.33.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+As a minor release, 0.6.33 is backward binary compatible with previous releases in the 0.6.x series.
+Libraries compiled with earlier versions can be used with 0.6.33 without change.
+0.6.33 is also forward binary compatible with 0.6.{29-32}, but not with earlier releases: libraries compiled with 0.6.33 cannot be used by projects using 0.6.{0-28}.
+
+Please report any issues [on GitHub](https://github.com/scala-js/scala-js/issues).
+
+## New warnings
+
+In Scala.js 0.6.33, the Scala.js will start reporting warnings when trying to override `equals` and `hashCode` in a JS type (extending `js.Any`).
```
Comment (gzm0) [inline _posts/news/2020-05-13-announcing-scalajs-0.6.33.md:38]: the Scala.js *compiler*?

**Pull request merged**